woocommerce/woocommerce-ios
# Domain selector: show attributed price info

Part of #8558 


## Description


This PR shows attributed price info for a paid domain suggestion on the domain selector. When there's a sale for the domain, the original price is shown in strikethrough style and the sale price is shown in yellow.

## Testing instructions


Prerequisite: a WC store with a WPCOM plan is required

- Log in if needed, and continue with a WPCOM store
- Go to the Menu tab
- Tap on the settings CTA
- Tap the `Domain` row
- Tap `Add a domain` or `Search for a domain` depending on if there are existing domains --> domain selector with paid domains should be shown, and each domain should have attributed price info as in the design VyLr7LvKodHE4kINfBE7Lw-fi-682%3A43394&t=SH7sZF4DTOOL8qjm-0
- Feel free to search for other domains --> each domain should have attributed price info

## Screenshots


light | dark
-- | --
![Simulator Screen Shot - iPhone 14 - 2023-01-30 at 11 53 11](https://user-images.githubusercontent.com/1945542/215384075-8694b805-623c-4e98-ac9a-9f4a0d359a69.png) | ![Simulator Screen Shot - iPhone 14 - 2023-01-30 at 11 53 23](https://user-images.githubusercontent.com/1945542/215384085-38528852-64bc-4429-90ff-e32c7b4f7f78.png)




---
